configure.ac

```diff
@@ -1118,7 +1118,7 @@ AS_IF([test "$ENABLE_IOSAPP" != "true" -a "$ENABLE_ANDROIDAPP" != "true"],
       [AC_MSG_CHECKING([for LibreOffice path])
       if test -n "$with_lo_path"; then
           # strip trailing '/' from LO_PATH, 'ln -s' with such path will otherwise fail
-          LO_PATH="${with_lo_path%/}"
+          LO_PATH=`readlink -f ${with_lo_path%/}`
```

Comment:
Would've been better if we wrapped the path in double-quotes. It's unlikely, but in the event that the path has spaces, this will break.